cluster-autoscaler/cloudprovider/mcm/mcm_manager.go

```diff
@@ -98,6 +100,9 @@ const (
 	machineDeploymentPausedReason = "DeploymentPaused"
 	// machineDeploymentNameLabel key for Machine Deployment name in machine labels
 	machineDeploymentNameLabel = "name"
+	// machinesMarkedByCAForDeletion is the annotation set by CA on machine deployment. Its value denotes the machines that
+	// CA marked for deletion by updating the priority annotation to 1 and scaling down the machine deployment.
+	machinesMarkedByCAForDeletion = "cluster-autoscaler.kubernetes.io/machines-marked-by-ca-for-deletion"
```

Comment:
```suggestion
	machinesMarkedByCAForDeletionAnnotation = "cluster-autoscaler.kubernetes.io/machines-marked-by-ca-for-deletion"
```
Need not be considered, but this is usually how annotations are defined